### ISSUE TITLE
sq-str and dq-str remove spaces from the quoted string

### ISSUE THREAD

**Issue by aesterline** [protoflex/parse-ez#3]
When using `sq-str` and `dq-str` the quoted string get spaces removed before being returned. 

Is the space stripping expected behavior?

``` clojure
user=> (p/parse p/sq-str "'Adam Esterline'")
"AdamEsterline"
user=> (p/parse p/dq-str "\"Adam Esterline\"")
"AdamEsterline"
```

Thanks for your help.

**Comment by protoflex** [comment]
This is a bug. As a temporary workaround, you can use the "no-trim" function like:
(p/parse #(p/no-trim p/sq-str "'Adam Esterline'"))

**Comment by protoflex** [comment]
Fixed in 0.3.3.
